### ISSUE TITLE
NEW FORK announcement

### ISSUE THREAD

**Issue by jchrys** [mirromutth/r2dbc-mysql#251]
## ⚠️ Deprecation Notice ⚠️
> As the original repository appears to be abandoned and we have been unable to reach the original maintainer, we regret to inform you that versions 0.8.2.RELEASE and earlier of dev.miku.r2dbc-mysql are now deprecated due to unresolved deprecation and vulnerability warnings.

# We Have Created a New Fork
You can find the new forked repository here:  https://github.com/asyncer-io/r2dbc-mysql 👈 

We've already resolved various bugs and updated dependencies to their latest versions. 
As a result, we have released version 0.9.0.

We've also implemented new features to improve the usability of the project. Our team is committed to maintaining and updating the project going forward.

We welcome anyone who is interested in contributing to this project, whether by fixing bugs, adding new features, or improving documentation. You can visit the repository today to try out the updated version of the project.

If you encounter any issues or have suggestions for improvements, please create an issue on the [GitHub repository](https://github.com/asyncer-io/r2dbc-mysql) or submit a pull request. 
We look forward to collaborating with you on this project!

# TL;DR
- A new fork of this project has been created at https://github.com/asyncer-io/r2dbc-mysql
- Version 0.9.0 has been released
- See below for instructions on how to use the new version.

## Maven

```xml

<dependency>
  <groupId>io.asyncer</groupId>
  <artifactId>r2dbc-mysql</artifactId>
  <version>0.9.0</version>
</dependency>
```

### Groovy DSL

```groovy
dependencies {
    // Maybe should to use `compile` instead of `implementation` on the lower version of Gradle.
    implementation 'io.asyncer:r2dbc-mysql:0.9.0'
}
```

### Kotlin DSL

```kotlin
dependencies {
    // Maybe should to use `compile` instead of `implementation` on the lower version of Gradle.
    implementation("io.asyncer:r2dbc-mysql:0.9.0")
}
```

**Comment by jchrys** [comment]
Thanks @mirromutth for pinning this issue. 😄

**Comment by jchrys** [comment]
🎉We've just  released the v`1.0.0` of `r2dbc-mysql`. 
This version fully supports the latest `r2dbc-spi`, providing improved performance and stability.
Please check out our repository: https://github.com/asyncer-io/r2dbc-mysql
